pkg/commands/entrypoints/entrypoints.go

```diff
@@ -88,6 +89,13 @@ func (c *EntrypointsCommand) Flags() *cli.FlagSet {
 
 	f := set.NewSection("COMMAND OPTIONS")
 
+	f.StringVar(&cli.StringVar{
+		Name:    "dir",
+		Target:  &c.flagDir,
+		Example: "-dir=terraform",
```

Comment:
Example doesnt usually include the flag name, just the value, throughout.

I believe a previous PR did this do, can you validate throughout?

Reply:
fixed.